src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -135,6 +135,22 @@ public void MissingPermissionsToReadEnvironmentVariable(SecurityException ex)
             }
         }
 
+        [NonEvent]
+        public void DroppedExportProcessorItems(long droppedCount, Type type)
+        {
+            if (this.IsEnabled(EventLevel.Informational, EventKeywords.All) && droppedCount <= 0)
```

Comment:
This might be faster:

```suggestion
            if (droppedCount <= 0 && this.IsEnabled(EventLevel.Informational, EventKeywords.All))
```

Reply:
My reasoning for doing in that order was just to have a similar look to the other functions, but ya I'll take this suggestion 👍